main.go

```diff
@@ -1,3 +1,144 @@
 package main
 
-func main() {}
+import (
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/microkit/command"
+	microserver "github.com/giantswarm/microkit/server"
+	"github.com/giantswarm/microkit/transaction"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+	"github.com/giantswarm/node-operator/flag"
+	"github.com/spf13/viper"
+
+	"github.com/giantswarm/node-operator/server"
+	"github.com/giantswarm/node-operator/service"
+)
+
+var (
+	description string     = "Project node-operator drains Kubernetes nodes on behalf of watched CRDs."
+	f           *flag.Flag = flag.New()
+	gitCommit   string     = "n/a"
+	name        string     = "node-operator"
+	source      string     = "https://github.com/giantswarm/node-operator"
+)
+
+func main() {
+	err := mainWithError()
+	if err != nil {
+		panic(fmt.Sprintf("%#v\n", microerror.Mask(err)))
+	}
+}
+
+func mainWithError() error {
+	var err error
+
+	// Create a new logger which is used by all packages.
+	var newLogger micrologger.Logger
+	{
+		loggerConfig := micrologger.DefaultConfig()
```

Comment:
Following the requested pattern in `cluster-operator`, maybe short `loggerConfig` to `c` or `lc` when it's only used within extra scope for config? If you agree with this, there are similar cases elsewhere in the code.

Reply:
Good point. Shortened.